.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
+
+- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/{{metadata['repo']['name']}}/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea.
+- [ ] Ensure the tests and linter pass.
+- [ ] Code coverage does not decrease (if any source code was changed.)
```

Comment:
Again, link to how to determine this. For us, though, I'm not sure we pay that much attention to this, nor is our tooling really in good shape to track/report this, so we might want to omit it.

Reply:
Removed.